alica_tests/CMakeLists.txt

```diff
@@ -24,12 +24,9 @@ find_package(alica_test_utility REQUIRED)
 find_package(alica_dynamic_loading REQUIRED)
 include_directories(include)
 
-include_directories(include src/test/Expr/include)
-file(GLOB_RECURSE autogen_SOURCES "src/test/Expr/src/*.cpp")
-
-include_directories(libalica-tests/include)
-file(GLOB_RECURSE AlicaTestLibSources "libalica-tests/src/*.cpp")
-add_library(alica-tests SHARED ${AlicaTestLibSources} ${autogen_SOURCES})
+include_directories(include libalica-tests/include)
```

Comment:
needs to be alica_tests/include (with an underscore)

Reply:
include should be enough, there is no alica_tests folder at the root level of the alica_tests package (alica_tests/alica_tests/include)